### PR TITLE
Sunil

### PR DESCRIPTION
**Pull request by sunil-ei**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,19 +1,21 @@
 {
-  "name": "marketplace-frontend",
+  "name": "kognitiv-trading-platform",
```
Comment (heeropunjabikogx) [inline package.json]: change this name to "marketplace-frontend"

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import 'antd/lib/button/style';
+import { withRouter } from 'react-router-dom';
+
+import '../assets/style/App.less'
+import { Dashboard } from './';
+
+class App extends React.Component {
```
Comment (heeropunjabikogx) [inline src/components/App.js]: we need a functional component

File: src/components/Sider.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Layout, Menu } from 'antd';
+import { Link } from "react-router-dom";
+import { createBrowserHistory } from "history";
+
+import Routes from '../routes/routes'
+
+class Sidebar extends React.Component {
+  render() {
```
Comment (heeropunjabikogx) [inline src/components/Sider.js]: need a functional component

File: package.json (hunk 1 above)
Comment (swapnil-kogx) [reply package.json]: Done.

File: src/components/App.js (hunk 2 above)
Comment (swapnil-kogx) [reply src/components/App.js]: Done.

**Pull request merged**